src/jdk.jpackage/share/classes/jdk/jpackage/internal/resources/HelpResources_de.properties

```diff
@@ -33,7 +33,7 @@ MSG_Help_win_install=\  --win-dir-chooser\n          Fügt ein Dialogfeld hinzu,
-MSG_Help_linux_install=\  --linux-package-name <Packagename>\n          Name für das Linux-Package, Standardwert: Anwendungsname\n  --linux-deb-maintainer <E-Mail-Adresse>\n          Maintainer für .deb-Package\n  --linux-menu-group <Menügruppenname>\n          Menügruppe, in der diese Anwendung abgelegt wird\n  --linux-package-deps\n          Erforderliche Packages oder Funktionen für die Anwendung\n  --linux-rpm-license-type <Typzeichenfolge>\n          Typ der Lizenz ("License: <Wert>" der RPM-SPEC-Datei)\n  --linux-app-release <Releasewert>\n          Releasewert der RPM-SPEC-Datei <Name> oder \n          Debian-Revisionswert der DEB-Kontrolldatei\n  --linux-app-category <Kategoriewert>\n          Gruppenwert der RPM-SPEC-Datei <Name> oder \n          Abschnittswert der DEB-Kontrolldatei\n  --linux-shortcut\n          Erstellt eine Verknüpfung für die Anwendung.\n
+MSG_Help_linux_install=\  --linux-package-name <Packagename>\n          Name für das Linux-Package, Standardwert: Anwendungsname\n  --linux-deb-maintainer <E-Mail-Adresse>\n          Maintainer für .deb-Package\n  --linux-menu-group <Menügruppenname>\n          Menügruppe, in der diese Anwendung abgelegt wird\n  --linux-package-deps <Paketabhängigkeit>[, <Paketabhängigkeit>...]\n          Erforderliche Packages oder Funktionen für die Anwendung\n  --linux-rpm-license-type <Typzeichenfolge>\n          Typ der Lizenz ("License: <Wert>" der RPM-SPEC-Datei)\n  --linux-app-release <Releasewert>\n          Releasewert der RPM-SPEC-Datei <Name> oder \n          Debian-Revisionswert der DEB-Kontrolldatei\n  --linux-app-category <Kategoriewert>\n          Gruppenwert der RPM-SPEC-Datei <Name> oder \n          Abschnittswert der DEB-Kontrolldatei\n  --linux-shortcut\n          Erstellt eine Verknüpfung für die Anwendung.\n
```

Comment:
`--linux-package-deps <package-dep-string>` to sync with HelpResources.properties